### ISSUE TITLE
About the training and testing results

### ISSUE THREAD

**Issue by whuhxb** [weiliu89/caffe#749]
Please use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) for usage, installation, or modeling questions, or other requests for help.
_Do not post such requests to Issues._ Doing so interferes with the development of Caffe.

Please read the [guidelines for contributing](https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md) before submitting this issue.

### Steps to reproduce

If you are having difficulty building Caffe or training a model, please ask the caffe-users mailing list. If you are reporting a build error that seems to be due to a bug in Caffe, please attach your build configuration (either Makefile.config or CMakeCache.txt) and the output of the make (or cmake) command.

### Your system configuration
Operating system:
Compiler:
CUDA version (if applicable):
CUDNN version (if applicable):
BLAS:
Python or MATLAB version (for pycaffe and matcaffe respectively):

I want to know whether the ssd_pascal.py is for training and the score_ssd_pascal.py is for testing. But for the ssd_pascal.py, the running results is shown as "Test_net: detection_eval:". Or the ssd_pascal.py can obtain the testing results?

**Comment by weiliu89** [comment]
It will periodically evaluate the test results on validation set during training time.

**Comment by whuhxb** [comment]
@weiliu89   You mean that the test results are obtained by score_ssd_pascal.py? Is that right?

**Comment by whuhxb** [comment]
@weiliu89  If the test results are obtained by score_ssd_pascal.py, the detection mAP obtained by finetuned SSD is only 66% on my own dataset, even lower than Faster R-CNN (ZF). Do you think that's acceptable? Or there are some strategies to improve the detection performance of SSD?

**Comment by whuhxb** [comment]
@weiliu89 If the test results are obtained by score_ssd_pascal.py, the detection mAP obtained by finetuned SSD is only 66% on my own dataset, even lower than Faster R-CNN (ZF) (70%). Do you think that's acceptable? Or there are some strategies to improve the detection performance of SSD?

**Comment by whuhxb** [comment]
@weiliu89  How to set the max_iter for score_ssd_pascal.py? You set as 0 for score_ssd_pascal.py, Can I change it to a lager number, such as 40000?

**Comment by fanfancy** [comment]
I found that results "Test_net: detection_eval:" in training process are the same as the one produced by running "score_ssd_xxx.py". Anyone knows the reason? Thanks.

**Comment by Betty-tkx** [comment]
@fanfancyWhat I understand is that score_ssd_xxx.py uses the last training model to test on the test set, while ssd_pascal. py periodically evaluates the results of the validation set.

**Comment by fanfancy** [comment]
@Betty-tkx Yes, I also think code should be behave as what you said. Thus the results should be different. But I can't get different results.

**Comment by Betty-tkx** [comment]
@fanfancy My results are almost the same, I would like to see the loss of validation set, but there is little progress.
